README.md

```diff
@@ -65,6 +65,8 @@ gpg --no-default-keyring --keyring "$KEYRING" --list-keys
 
 echo "deb [signed-by=$KEYRING] https://deb.nodesource.com/$VERSION $DISTRIBUTION main" | sudo tee /etc/apt/sources.list.d/nodesource.list
 echo "deb-src [signed-by=$KEYRING] https://deb.nodesource.com/$VERSION $DISTRIBUTION main" | sudo tee -a /etc/apt/sources.list.d/nodesource.list
+
+sudo apt-get update && sudo apt-get install nodejs
```

Comment:
```suggestion
sudo apt update && sudo apt install nodejs
```

Reply:
I thought about that as well but then used the strings from `https://deb.nodesource.com/` as i wasn't sure if there is a reason why they used `apt-get` vs. just `apt`.

Reply:
Not sure why they still use apt-get. I tend to replace it with just apt since some years.